src/app/containers/CpsAssetPageMain/index.jsx

```diff
@@ -15,6 +15,7 @@ import cpsAssetPagePropTypes from '../../models/propTypes/cpsAssetPage';
 
 const componentsToRender = {
   headline: headings,
+  subheadline: headings,
```

Comment:
issue is for both subeadings and crossheadings - happy for it to be seperate though (if you amend the description)

Reply:
Those get converted [here](https://github.com/bbc/simorgh/pull/4495/files#diff-c371abcd57f93ea8f44152e1a61d997fR11-R13) :) By the time they get to `CpsAssetPageMain` theyre all `subheadline` :)

Reply:
never mind - realised you're changing all the types to subheading in the preprocessor - thought we would want to maintain the original type?  This is fine though